src/main/resources/application.properties

```diff
@@ -1,5 +1,5 @@
 aad.clientId=Enter_the_Application_Id_here
-aad.authority=https://login.microsoftonline.com/Enter_the_Tenant_Info_Here/
```

Comment:
Although I agree "Enter_the_Tenant_Id_Here" is more accurate,  this needs to be "Enter_the_Tenant_Info_Here" because that's what the portal team looks for when do they do the auto configuration for the quickstart. More details here: https://microsoft.sharepoint.com/:w:/t/Identity-DevEx/EZskrq88i0RPmlb1ykW9uqEBX7oxWNuLglFhIX0twzagCw?e=X0tlJS in section 4.2

Reply:
Oh ok thanks for the document. Yes as an end user I too felt "Tenant_Id" would be more clear and straight forward.

Reply:
So if it is mandatory I will change it to "Tenant_Info"

Reply:
I think the reason for the Tenant_Info is that it could be Tenant_Id or Tenant name or common etc depending on the audience the app supports. Anyway, as Santiago mentions if we change this string the portal integration will break so we need to use Tenant_Info.

Reply:
ok will change it back to Tenant_Info